ftrip.io.catalog-service/ftrip.io.catalog-service/Accommodations/UseCases/CreateAccommodation/CreateAccommodationRequestValidator.cs

```diff
@@ -45,7 +52,8 @@ public CreateAccommodationRequestValidator()
                 .InclusiveBetween(0, 24);
 
             RuleFor(request => request.CheckInTo - request.CheckInFrom)
-                .GreaterThanOrEqualTo(2);
+                .GreaterThanOrEqualTo(2)
+                .WithMessage("The check-in window should be at least 2 hours");
```

Comment:
It would be nice to put all validation messages Strings.res. I have an example of that in UserService.